client/utils/auth.py

```diff
@@ -38,9 +35,11 @@
 REDIRECT_PORT = 6165
 
 TIMEOUT = 10
-SERVER = 'https://okpy.org'
+
 INFO_ENDPOINT = "https://www.googleapis.com/oauth2/v1/userinfo"
```

Comment:
Shouldn't this be the OK userinfo endpoint? The okpy access token won't work with Google.